lib/modules/datasource/galaxy-collection/index.ts

```diff
@@ -21,7 +16,7 @@ export class GalaxyCollectionDatasource extends Datasource {
 
   override readonly customRegistrySupport = false;
```

Comment:
```suggestion
  override readonly customRegistrySupport = true;
```

Maybe no harm in this?